app/src/test/java/com/nervousfish/nervousfish/service_locator/ServiceLocatorNoNetworkTest.java

```diff
@@ -0,0 +1,154 @@
+package com.nervousfish.nervousfish.service_locator;
+
+import com.nervousfish.nervousfish.modules.constants.IConstants;
+import com.nervousfish.nervousfish.modules.cryptography.IEncryptor;
+import com.nervousfish.nervousfish.modules.cryptography.IKeyGenerator;
+import com.nervousfish.nervousfish.modules.database.IDatabase;
+import com.nervousfish.nervousfish.modules.filesystem.IFileSystem;
+import com.nervousfish.nervousfish.modules.pairing.IBluetoothHandler;
+import com.nervousfish.nervousfish.modules.pairing.INfcHandler;
+import com.nervousfish.nervousfish.modules.pairing.IQRHandler;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+
+public final class ServiceLocatorNoNetworkTest {
```

Comment:
Shouldn't `ServiceLocatorNoNetwork` a test class itself, i.e. not part of the source code?